tools/codegen/api/python.py

```diff
@@ -228,7 +228,7 @@ class PythonArgument:
     # Compute argument formal for python argument parsing.
     # Needs to be consistent with torch/csrc/utils/python_arg_parser.h.
     def argument_str(self, *, method: bool = False) -> str:
-        type_str = argument_type_str(self.type)
+        type_str = argument_type_str(self.type).replace('const ', '').replace(' &', '')
```

Comment:
hmm....

Reply:
Agreed. The reason this is required is that the python arg parser tokenizes types with spaces wrongly, it only supports typenames without spaces. So the correct way of fixing this would be to fix the tokenizer in PythonArgParser.